pages/vi/vi-track-progress.html

```diff
@@ -69,7 +69,7 @@
 		console.log(object)
 	
 
-	
+if(user == "" || user == null) 	{	} else {
```

Comment:
I understand that the original script is not formatted well. But I think we could at least be consistent with having an empty space between`if` and `(`. Keeping code style consistent is also essential when we work on a group project.

Leaving the if block empty looks a bit weird.
Would using `if (!(user == "" || user == null))` without `else` be better? Or alternatively use your original if condition with `return;` right after [line 64](https://github.com/yaoxing1990/yaoxing1990.github.io/blob/9ad11852704ddd589848754021012abb561ca4e1/pages/vi/vi-track-progress.html#L64) etc.